zlibstubs.c:171:33: error: use of undeclared identifier 'uint32'

I've followed all instructions from https://github.com/facebook/pfff/blob/master/install_macos.txt on OS X El Capitan Version 10.11.5. Getting the following error when running make.

```
$: make
/Applications/Xcode.app/Contents/Developer/usr/bin/make rec
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C commons
make[2]: Nothing to be done for `all'.
/Applications/Xcode.app/Contents/Developer/usr/bin/make all -C external/ocamlgraph && /Applications/Xcode.app/Contents/Developer/usr/bin/make -C commons/graph
make[2]: Nothing to be done for `all'.
make[2]: Nothing to be done for `all'.
/Applications/Xcode.app/Contents/Developer/usr/bin/make all -C external/ocamlgtk && /Applications/Xcode.app/Contents/Developer/usr/bin/make gui -C commons
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C src all
make[3]: Nothing to be done for `all'.
/Applications/Xcode.app/Contents/Developer/usr/bin/make INCLUDESEXTRA="-I ../external/ocamlgtk/src" commons_gui.cma
make[3]: `commons_gui.cma' is up to date.
/Applications/Xcode.app/Contents/Developer/usr/bin/make features -C commons
make[2]: Nothing to be done for `features'.
set -e; for i in commons commons/graph external/jsonwheel external/ocamlgraph external/ocamlgtk external/ocamlcairo external/ocamlzip external/extlib external/ptrees external/javalib/src globals h_version-control h_visualization h_files-format h_program-lang graph_code h_program-visual matcher lang_ml/parsing lang_ml/analyze  lang_nw/parsing lang_nw/analyze lang_lisp/parsing lang_lisp/analyze lang_haskell/parsing lang_haskell/analyze lang_php/parsing lang_php/matcher lang_php/pretty lang_sql/parsing lang_js/parsing lang_js/analyze lang_cpp/parsing lang_cpp/analyze lang_c/parsing lang_c/analyze lang_clang/parsing lang_clang/analyze lang_java/parsing lang_java/analyze lang_bytecode/parsing lang_bytecode/analyze lang_python/parsing lang_python/analyze lang_csharp/parsing lang_csharp/analyze lang_rust/parsing lang_rust/analyze lang_opa/parsing lang_opa/analyze lang_erlang/parsing lang_erlang/analyze lang_php/analyze lang_php/analyze/foundation lang_php/analyze/checker lang_php/analyze/tools lang_php/analyze/qa_test lang_html/parsing lang_html/analyze lang_css/parsing lang_web/parsing lang_text mini code_map code_graph demos; do /Applications/Xcode.app/Contents/Developer/usr/bin/make -C $i all || exit 1; done
make[2]: Nothing to be done for `all'.
make[2]: Nothing to be done for `all'.
make[2]: Nothing to be done for `all'.
make[2]: Nothing to be done for `all'.
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C src all
make[3]: Nothing to be done for `all'.
/Applications/Xcode.app/Contents/Developer/usr/bin/make -C src all
make[3]: Nothing to be done for `all'.
ocamlc -g -c -ccopt -g -ccopt -I/usr/local/include -ccopt -I/usr/local/lib/ocaml zlibstubs.c
zlibstubs.c:171:33: error: use of undeclared identifier 'uint32'
  return caml_copy_int32(crc32((uint32) Int32_val(crc),
                                ^
1 error generated.
make[2]: *** [zlibstubs.o] Error 2
make[1]: *** [rec] Error 1
make: *** [all] Error 2
```

--- a/external/ocamlzip/zlibstubs.c
+++ b/external/ocamlzip/zlibstubs.c
@@ -16,6 +16,7 @@
 /* Stub code to interface with Zlib */
 
 #include <zlib.h>
+#include <stdint.h>
 
 #include <caml/mlvalues.h>
 #include <caml/alloc.h>
@@ -168,7 +169,7 @@ value camlzip_inflateEnd(value vzs)
 
 value camlzip_update_crc32(value crc, value buf, value pos, value len)
 {
-  return caml_copy_int32(crc32((uint32) Int32_val(crc), 
+  return caml_copy_int32(crc32((uint32_t) Int32_val(crc), 
                           &Byte_u(buf, Long_val(pos)),
                           Long_val(len)));
 }

